### PR TITLE
Merge upstream

### PR DESCRIPTION
**Pull request by richard-cox**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,32 +2,19 @@
 
 ## 4.1.0
 
-[Full Changelog](https://github.com/SUSE/stratos/compare/4.0.1...4.1.0)
+[Full Changelog](https://github.com/cloudfoundry/stratos/compare/4.0.1...4.1.0)
```
Comment (richard-cox) [inline CHANGELOG.md]: revert

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -6,10 +6,10 @@ There are two main forms of contribution: reporting issues and performing code c
 
 ## Reporting Issues
 
-If you find a problem with Stratos, report it using [GitHub issues](https://github.com/suse/stratos/issues/new).
```
Comment (richard-cox) [inline CONTRIBUTING.md]: revert

File: README.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
 # Stratos
 
-<a href="https://travis-ci.com/suse/stratos/branches"><img src="https://travis-ci.com/suse/stratos.svg?branch=master"></a>&nbsp;<a style="padding-left: 4px" href="https://codeclimate.com/github/suse/stratos/maintainability"><img src="https://api.codeclimate.com/v1/badges/61af8b605f385e894632/maintainability" /></a>
```
Comment (richard-cox) [inline README.md]: revert

File: angular.json [hunk 4]
```diff
@@ -349,34 +349,6 @@
         }
       }
     },
-    "extensions": {
```
Comment (richard-cox) [inline angular.json]: revert

File: deploy/ci/suse-console-dev-releases.yml [hunk 5]
```diff
@@ -1,231 +0,0 @@
-# This pipeline creates an Alpha, Beta or RC release
```
Comment (richard-cox) [inline deploy/ci/suse-console-dev-releases.yml]: revert

File: deploy/kubernetes/custom/customize-helm.sh [hunk 6]
```diff
@@ -1,77 +0,0 @@
-#!/bin/bash
```
Comment (richard-cox) [inline deploy/kubernetes/custom/customize-helm.sh]: revert, along with deploy/kubernetes/custom/icon.png

File: deploy/suse/patches/nginx/Dockerfile [hunk 7]
```diff
@@ -1,4 +0,0 @@
-FROM registry.suse.com/cap/stratos-console:2.7.0-35f5964bd-cap
```
Comment (richard-cox) [inline deploy/suse/patches/nginx/Dockerfile]: revert these three (deploy/suse/*)

File: src/tsconfig.json [hunk 8]
```diff
@@ -38,7 +38,6 @@
       "@stratosui/cloud-foundry": ["frontend/packages/cloud-foundry/src/public_api.ts"],
       "@stratosui/cf-autoscaler": ["frontend/packages/cf-autoscaler/src/public_api.ts"],
       "@example/extensions": ["frontend/packages/example-extensions/src/public-api.ts"],
-      "@susestratos/extensions": ["frontend/packages/suse-extensions/src/public-api.ts"],
```
Comment (richard-cox) [inline src/tsconfig.json]: revert, along with stratos.yaml

File: deploy/tools/populate-cf/susecon-2019.sh [hunk 9]
```diff
@@ -1,320 +0,0 @@
-#!/bin/bash
```
Comment (richard-cox) [inline deploy/tools/populate-cf/susecon-2019.sh]: revert

File: docs/suse/deploy.md [hunk 10]
```diff
@@ -1,175 +0,0 @@
-# Deploying Stratos
```
Comment (richard-cox) [inline docs/suse/deploy.md]: revert these, docs/suse/*

File: package.json [hunk 11]
```diff
@@ -28,7 +27,6 @@
     "test-frontend:store": "NG_TEST_SUITE=store ng test store --code-coverage --watch=false",
     "test-frontend:cloud-foundry": "NG_TEST_SUITE=cloud-foundry ng test cloud-foundry --code-coverage --watch=false",
     "test-frontend:cf-autoscaler": "NG_TEST_SUITE=autoscaler ng test cf-autoscaler --code-coverage --watch=false",
-    "test-frontend:extensions": "NG_TEST_SUITE=extensions ng test extensions --code-coverage --watch=false",
```
Comment (richard-cox) [inline package.json]: revert

File: stratos.yaml [hunk 12]
```diff
@@ -1,2 +0,0 @@
-title: SUSE Stratos Console
```
Comment (richard-cox) [inline stratos.yaml]: revert

File: src/test-e2e/e2e.ts [hunk 13]
```diff
@@ -20,8 +20,8 @@ export class E2E {
    * Temporary location for customization, we should in future look to fetch this a better way from client side code
   */
   public static customization: CustomizationsMetadata = {
-    alwaysShowNavForEndpointTypes: (epType) => false // Should match implementation in suse.module
-  }
+    alwaysShowNavForEndpointTypes: (epType) => true
```
Comment (richard-cox) [inline src/test-e2e/e2e.ts]: revert?

**Review by richard-cox — COMMENTED** [review]
see comments

**Review by nwmac — APPROVED** [review]
LGTM

**Pull request merged**